Source/MonoGame.Extended/Collections/Poolable.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace MonoGame.Extended.Collections
+{
+    public abstract class Poolable : IPoolable
```

Comment:
I think I'd rather not have this class and just provide the interface. That way people are forced to think about the `ResetState` method. If they derive from this class and forget to implement ResetState things could get hairy.